config/bugsnag.php

```diff
@@ -0,0 +1,115 @@
+ * file that was distributed with this source code.
+ */
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | API Key
+    |--------------------------------------------------------------------------
+    |
+    | You can find your API key on your Bugsnag dashboard.
+    |
+    | This api key points the Bugsnag notifier to the project in your account
+    | which should receive your application's uncaught exceptions.
+    |
+    */
+
+    'api_key' => '',
+
+    /*
+    |--------------------------------------------------------------------------
+    | Notify Release Stages
+    |--------------------------------------------------------------------------
+    |
+    | Set which release stages should send notifications to Bugsnag.
+    |
+    | Example: array('development', 'production')
+    |
+    */
+
+    'notify_release_stages' => ['production'],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Endpoint
+    |--------------------------------------------------------------------------
+    |
+    | Set what server the Bugsnag notifier should send errors to. By default
+    | this is set to 'https://notify.bugsnag.com', but for Bugsnag Enterprise
+    | this should be the URL to your Bugsnag instance.
+    |
+    */
+
+    'endpoint' => 'https://beacon.cachethq.io/notify/',
+
+    /*
+    |--------------------------------------------------------------------------
+    | Filters
+    |--------------------------------------------------------------------------
+    |
+    | Use this if you want to ensure you don't send sensitive data such as
+    | passwords, and credit card numbers to our servers. Any keys which
+    | contain these strings will be filtered.
+    |
+    */
+
+    'filters' => ['password'],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Callbacks
+    |--------------------------------------------------------------------------
+    |
+    | Enable this if you'd like us to enable our default set of notification
+    | callbacks. These add things like the cookie information and session
+    | details to the error to be sent to Bugsnag.
+    |
+    | If you'd like to add your own callbacks, you can call the
+    | Bugsnag::registerCallback method from the boot method of your app
+    | service provider.
+    |
+    */
+
+    'callbacks' => true,
+
+    /*
+    |--------------------------------------------------------------------------
+    | User
+    |--------------------------------------------------------------------------
+    |
+    | Enable this if you'd like us to set the current user logged in via
+    | Laravel's authentication system.
+    |
+    | If you'd like to add your own user resolver, you can call the
+    | Bugsnag::registerUserResolver method from the boot method of your app
+    | service provider.
+    |
+    */
+
+    'user' => true,
```

Comment:
I don't think we should do this. Since users from several status pages would be grouped into one on Bugsnag.